huffman/test/decode_test.cpp

```diff
@@ -0,0 +1,4 @@
+namespace gpu_deflate::human {
+// encoded data from dahuffman readme.rst, but in hex.
+// ['86', '7c', '25', '13', '69', '40']
```

Comment:
I thought we got something working last week but maybe I'm mistaken? Or do you have local changes that haven't been pushed?

Reply:
No I don't have anything locally.